build-files/trimal/1.2/Dockerfile

```diff
@@ -0,0 +1,50 @@
+##### ----- STAGE 1: app ----- #####
+FROM ubuntu:focal AS app
+
+# Set software version
+ARG TRIMAL_VER="v1.2rev59"
+
+# Metadata labels
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="TrimAl"
+LABEL software.version="${TRIMAL_VER}"
+LABEL description="trimAl is a tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment."
+LABEL website="https://vicfero.github.io/trimal/"
+LABEL license="https://github.com/inab/trimal?tab=GPL-3.0-1-ov-file#readme"
+LABEL maintainer="FirstName LastName"
+LABEL maintainer.email="my.email@email.com"
+
+# Install build dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    wget \
+    ca-certificates && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# Download and build TrimAl
+WORKDIR /opt
+RUN wget https://github.com/user-attachments/files/17989858/trimal.v1.2rev59.tar.gz -O trimal.tar.gz && \
```

Comment:
Can you use the TRIMAL_VER variable here? It helps when we copy files to get updated images.